gravityflow/gravityflowstripe
# Process capture for PaymentIntents created using the Stripe Card field

re: [HS#14446](https://secure.helpscout.net/conversation/1256072874/14446/)

This updates `Gravity_Flow_Step_Stripe_Capture_Charge::capture_payment_intent()` to trigger `GF_Stripe::process_capture()` when capturing a PaymentIntent created when using the Stripe Card field.

This fixes an issue where the entry is not updated with the payment details which can impact subsequent steps which depend on the entry containing those new details.

## Testing Instructions
1. Configure Stripe Add-On to use the Stripe Card field as the payment collection method
1. Import form: https://www.dropbox.com/s/u918z55zh8j91k5/gravity-flow-stripe-capture.json?dl=0
1. Create product/service type Stripe feed for imported form
1. Preview and submit form with test card e.g. 4242424242424242
1. View Gravity Forms entry detail page to confirm payment status is Authorized
1. Approval entry
1. Find capture step has added note indicating authorised payment was captured
1. View Gravity Forms entry detail page to find the payment status is still Authorized
1. Switch to this branch
1. Repeat steps 4-8 to find that when the capture step runs the payment status does change to Paid
